hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieMultiTableDeltaStreamer.java

```diff
@@ -442,17 +443,11 @@ public void sync() {
   }
 
   public static class Constants {
```

Comment:
this class ideally should be private. but for tests, let's make it and its members all package access?

Reply:
Addressed.